master/buildbot/test/util/steps.py

```diff
@@ -51,9 +51,12 @@ def _dict_diff(d1, d2):
 
     for k in both:
         if isinstance(d1[k], dict) and isinstance(d2[k], dict):
-            missing_in_v1, missing_in_v2, different_in_v = _dict_diff(d1[k], d2[k])
-            missing_in_d1.extend(['{0}.{1}'.format(k, m) for m in missing_in_v1])
-            missing_in_d2.extend(['{0}.{1}'.format(k, m) for m in missing_in_v2])
+            missing_in_v1, missing_in_v2, different_in_v = _dict_diff(
```

Comment:
not sure if those are really needed. I usually cut at 100. cutting at 72 will mean a lot of such refacto, and to me loss of readability.